src/pages/profile/components/profile/PostList.tsx

```diff
@@ -13,22 +13,26 @@ const PostListTitle = styled.h1`
   ${tw`text-lg font-bold mb-4`}
 `
 
-const LikedPostList = ({ posts, listTitle }: PostListProps) => {
+const PostList = ({ posts, listTitle }: PostListProps) => {
   return (
     <PostListContainer>
       <PostListTitle>{listTitle}</PostListTitle>
       <section className="flex flex-col gap-2">
-        {posts?.map((post, index) => (
-          <button key={post._id}>
-            <PostItem
-              postDetail={post}
-              index={index}
-            />
-          </button>
-        ))}
+        {posts?.length ? (
+          posts?.map((post, index) => (
```

Comment:
P5 : 21번째 줄에서 옵셔널체이닝으로 null 체크도 동시에 해주고 있어서  22번째 줄에 옵셔널 체이닝은 불필요한거 같습니다! 
```suggestion
posts.map((post, index) => (
```